File: Codechef/Penalty/Penalty.py
T = int(input())
while T:
    T -= 1
    A = list(map(int, input().split()))
    odd = [A[i] for i in range(len(A)) if (i % 2 == 1)]
    even = [A[i] for i in range(len(A)) if (i % 2 == 0)]
    odd_one_count = [i for i in odd if i == 1]
    even_one_count = [i for i in even if i == 1]
    if (odd_one_count > even_one_count):
        print(2)
    elif (odd_one_count < even_one_count):
        print(1)
    else:
        print(0)

File: Codechef/Penalty/Penalty.cpp
#include <iostream>
using namespace std;

int main()
{
    int T;
    cin >> T;
    while (T--)
    {
        int A[10];
        for (int i = 0; i < 10; i++)
        {
            cin >> A[i];
        }
        int odd[5], even[5];
        for (int i = 0; i < 10; i++)
        {
            if (i % 2 == 1)
            {
                odd[i] = A[i];
            }
            else if (i % 2 == 0)
            {
                even[i] = A[i];
            }
        }
        int odd_one_count[5], even_one_count[5];
        
    }
    return 0;
}